libkbfs/bare_root_metadata_v3.go

```diff
@@ -88,16 +88,19 @@ type BareRootMetadataV3 struct {
 	// of writing to the given folder.
 	FinalizedInfo *tlf.HandleExtension `codec:"fi,omitempty"`
 
-	// The sequence number of the global Keybase Merkle tree at the
-	// time this update was created (from the writer's perspective).
-	// This field was added to V3 after it was live for a while, and
-	// older clients that don't know about this field yet might copy
-	// it into new updates via the unknown fields copier. Which means
-	// new MD updates might end up referring to older Merkle roots.
-	// That's ok since this is just a hint anyway, and shouldn't be
-	// fully trusted when checking MD updates against the Merkle tree.
+	// DEPRECATED -- see KBMerkleRoot.Seqno instead.
```

Comment:
Yeah fair point.  I was originally thinking that, if we were going to ignore the merkle stuff during ID computation, it would be best to keep it all in a single field.  Though one other advantage to this way is that if other fields are added to `MerkleRootV2` in the future, we'll get them for free.

But actually, I don't think I need to deprecate the field at all; I can just nuke it.  The unknown fields should take care of it.  Does that sound ok?